css/index.less

```diff
@@ -356,6 +356,7 @@ ul.tags {
   }
   > .dropdown {
     overflow: visible;
+    max-height: 31vh;
```

Comment:
as far as i can tell, this doesn't have any effect on the size of the `ul.dropdown-menu` child, whose max-height is set in dropdown.less